src/main.rs

```diff
@@ -202,6 +203,10 @@ async fn run(miniserve_config: MiniserveConfig) -> Result<(), ContextualError> {
             .app_data(stylesheet.clone())
             .wrap_fn(errors::error_page_middleware)
             .wrap(middleware::Logger::default())
+            .wrap(Condition::new(
+                miniserve_config.compress_response_enabled,
```

Comment:
Let's be consistent with ourselves here and remove the extra import at the top.
```suggestion
            .wrap(middleware::Condition::new(
                miniserve_config.compress_response_enabled,
```